File: frontend/tramonto-security/src/stores/dtos/UserDto.ts
import BaseDto from 'stores/dtos/Base.dto';
import Role from 'stores/dtos/Role';

export class UserDto implements BaseDto {
  email = '';

  firstName = '';

  lastName = '';

  roles: Array<Role> = [];

  check = false;

  createdAt = '';

  id = '';

  status = '';

  updatedAt = '';
}

File: backend/tramonto/src/main/java/br/com/wsss/tramonto/dto/input/UserDto.java
package br.com.wsss.tramonto.dto.input;

import java.util.Set;

import com.fasterxml.jackson.annotation.JsonProperty;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class UserDto extends BaseDto {

	private String firstName;
	private String lastName;
	private String email;
	private Set<RoleDto> roles;
	
	@JsonProperty(access = JsonProperty.Access.WRITE_ONLY)
	private String password;

}
